trainee_template/explanation.amlg

```diff
@@ -883,4 +883,13 @@
 			)
 		)
 	)
+
+	;Get the appropriate empty data structure for a given detail
+	#GetDetailDataStructure
+	(if (contains_value (list "categorical_action_probabilities") detail)
```

Comment:
if it's a list of a single value why are you doing (contains_value) ?  should this be hardcoded or refer to the trainee attribute?

Reply:
the Trainee attribute doesn't specify which details use what structure, so I can't do that. The only reason it's a list is because this should get larger in the future.

But I do realize how silly it looks now... I'll just make it an (=) for now.